src/Paket.Core/Versioning/FrameworkHandling.fs

```diff
@@ -919,6 +923,7 @@ type FrameworkIdentifier =
         | DotNetFramework FrameworkVersion.V4_7_1 -> [ DotNetFramework FrameworkVersion.V4_7; DotNetStandard DotNetStandardVersion.V2_0 ]
         | DotNetFramework FrameworkVersion.V4_7_2 -> [ DotNetFramework FrameworkVersion.V4_7_1 ]
         | DotNetFramework FrameworkVersion.V4_8 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
+        | DotNetFramework FrameworkVersion.V4_8_1 -> [ DotNetFramework FrameworkVersion.V4_7_2 ]
```

Comment:
I think there is a missing fallback to 4_8 here.

Reply:
rather, 4_8 should replace 4_7_2 here

Reply:
Changed as requested. Tough I do not fully understand, what that functions intent is…